docs/immortal-empires/tomb-kings.md

```diff
@@ -0,0 +1,27 @@
+# Tomb Kings
+
+## Arkhan the Black: Followers of Nagash
+
+* **Route I - Thy Kingdom Come**
+  * Arkhan has feigned obedience to Settra for long enough. The time has come to march out and retake the ancient lands of his master, the Great Necromancer Nagash.
+  * Occupy at least 6 provinces:
+    * Land of Assassins, Coast of Araby, Great Desert of Araby, The Cracked Land, Great Mortis Delta, Land of the Dead, Shifting Sands, Land of the Dervishes
+  * Destroy faction:
+    * Khemri
+  * **Reward**:
+    * _Thy Will Be Done_: Recruitment rank +4, Unit capacity: %+4\nTomb Guard\nTomb Guard (Halberds)\nSkeleton Chariot\nSkeleton Archer Chariot
+
+* **Route II - Cursed to Serve**
+  * The vampires did not drink from the Elixir of Life as Arkhan did, but from a lesser, corrupted version that cursed them with unlife. They are forever bound to the will of Nagash, and as his foremost servant, Arkhan can command them just as easily.
+  * Occupy at least 4 settlements:
+    * Galleon's Graveyard, The Awakening, Lahmia, Castle Drakenhof, Silver Pinnacle, Blackstone Post, Haunted Forest
+  * **Reward**:
+    * _Know Your Place_: Vampire Count and Vampire Coast allegiance +100%, Vampiric corruption +5 (factionwide), Vampiric corruption adjacent provinces +5 (factionwide)
```

Comment:
I think "factionwide" is irrelevant here. Just specify when it is something specific, like a particular area.

Reply:
I would also go with something like 6/4. I think local should also be stronger than adjacent corruption.

Reply:
Yes to 6/4.

Re factionwide:
1 I had an issue where if I put _unseen after the entire reward disappeared.
2 If you take away factionwide player loses info. Is the corruption coming from Mannfred, from only those regions, from factionwide? Remove it if you want but player loses information.

Reply:
By default, all rewards regarding Income, Corruption, Public Order or Growth are factionwide. At the same time, by default all unit boosts apply all armies. I think there is no research that gives you "Public Order: +2 (Factionwide)" or "Leadership: +6 when fighting against Lizardmen (All armies)". It is the default value.

I would only specify if you target individual lords, provinces or regions.

Reply:
Change has been made